pkg/stdlib/actions/v2/op.cue

```diff
@@ -0,0 +1,101 @@
+import (
```

Comment:
Because I defined the stable version, `vela/op` is also `v1`, and 'v2' in the dir is just for test.
But now the default version has a new action 'config' but 'v2' doesn't. The two versions is different now.

Reply:
I don't think that we need v2 here since there're no compatible issues between v1 and v2.